### PR TITLE
Fixes #208

### LINKED ISSUE #208: We can only have numbers that start from 1 and not from 0 in RakuScript
Now we use `-?[1-9][0-9\\.]+` so we can only use numbers that 
start from 1 and not 0 like this test from <https://regexr.com> shows:
image

Also we have it declared 2 times:
image
So we should remove one them.

But also like @theludovyc sayed:
> Yeah I do not use TOKEN but I keep them. Is dead code. 

So maybe we should remove Tokens ?

### PR DESCRIPTION
**Pull request by Jeremi360**
fixes #208

### CODE REVIEW

File: addons/Rakugo/lib/systems/Parser.gd [hunk 1]
```diff
@@ -35,12 +35,12 @@ var Tokens := {
...
 	TOKEN_VARIABLE = "^[a-zA-Z_][a-zA-Z_0-9]+",
 	TOKEN_TAB = "^\\t",
-	TOKEN_NUMERIC = "-?[1-9][0-9.]*"
+	TOKEN_NUMERIC = "-?[1-9][0-9\\.]*",
 }
 
 var Regex := {
 	NAME = "[a-zA-Z][a-zA-Z_0-9]*",
-	NUMERIC = "-?[1-9][0-9.]*",
+	NUMERIC = "-?[1-9][0-9\\.]*",
```
Comment (theludovyc) [inline addons/Rakugo/lib/systems/Parser.gd]: With this regex, you can't write a number like "0.5". But you can write "1.1.1"

File: addons/Rakugo/lib/systems/Parser.gd [hunk 2]
```diff
@@ -35,12 +35,12 @@ var Tokens := {
 	TOKEN_STRING = "^\".*?\"",
 	TOKEN_VARIABLE = "^[a-zA-Z_][a-zA-Z_0-9]+",
 	TOKEN_TAB = "^\\t",
-	TOKEN_NUMERIC = "-?[1-9][0-9.]*"
+	TOKEN_NUMERIC = "-?[1-9][0-9\\.]*",
```
Comment (theludovyc) [inline addons/Rakugo/lib/systems/Parser.gd]: If you are agree, we can remove all tokens :)

**Review by theludovyc — CHANGES_REQUESTED** [review]

File: addons/Rakugo/lib/systems/Parser.gd (hunk 1 above)
Comment (Jeremi360) [reply addons/Rakugo/lib/systems/Parser.gd]: Strange I sure that I change it to 1 to 0, maybe I pressed ctrl-z by mistake before commit or something.
I will fix it soon

File: addons/Rakugo/lib/systems/Parser.gd (hunk 2 above)
Comment (Jeremi360) [reply addons/Rakugo/lib/systems/Parser.gd]: If we really don't use it then, I will remove them, but in speared PR / issue to keep things clean

File: addons/Rakugo/lib/systems/Parser.gd (hunk 1 above)
Comment (Jeremi360) [reply addons/Rakugo/lib/systems/Parser.gd]: fixed with my last commit.
Yes I tested my new regex you can check it here: https://regexr.com/7d0d4 @theludovyc

File: addons/Rakugo/lib/systems/Parser.gd (hunk 2 above)
Comment (theludovyc) [reply addons/Rakugo/lib/systems/Parser.gd]: I am agree with you :) !

File: addons/Rakugo/lib/systems/Parser.gd (hunk 1 above)
Comment (theludovyc) [reply addons/Rakugo/lib/systems/Parser.gd]: This link with some test is so perfect 👌 !

**Review by theludovyc — APPROVED** [review]

**Comment by theludovyc** [comment]
Ready for merge :) !

**Pull request merged**